plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSourceProvider.java

```diff
@@ -754,6 +761,11 @@ else if (column.isRowPositionColumn()) {
                     constantPopulatingPageSourceBuilder.addDelegateColumn(parquetSourceChannel);
                     parquetSourceChannel++;
                 }
+                else if (column.getId() == FILE_PATH.id()) {
```

Comment:
Two nitpicks: Can you move this up next to the other `addConstantColumn` blocks? I was trying to keep all the constants at the top and delegate columns at the bottom.

Other nit, can you add a `column.isPathColumn()` method on `IcebergColumnHandle` rather than checking the id's here?

Reply:
done @alexjo2144